robobase/envs/bigym.py

```diff
@@ -253,21 +256,31 @@ def _demo_to_steps(
 
         for demo in demo_list:
             cur_demo = []
+            last_timestep = False
+            
+            # Detect whether this demo is successful or not
+            rewards = []
+            for step in demo:
+                reward = step.reward
+                rewards.append(reward)
+            successful_demo = sum(rewards) > 0.25
+            
             for i, step in enumerate(demo):
-                step.info.update({"demo": 1})
+                step.info.update({"demo": int(successful_demo)})
```

Comment:
wait, so not all demo is successful?

Reply:
Have you tried this branch: https://github.com/chernyadev/bigym/tree/fix_original_demos? Previously some demos are broken due to an upgrade of mujoco version

Reply:
I tried new branch but many demos still fail

Reply:
ok let's live with this for now and perhaps we need @chernyadev 's help to fix this.